linera-service/src/server.rs

```diff
@@ -473,6 +471,8 @@ fn log_file_name_for(command: &ServerCommand) -> Cow<'static, str> {
 }
 
 async fn run(options: ServerOptions) {
+    linera_base::tracing::init_with_opentelemetry(&log_file_name_for(&options.command)).await;
```

Comment:
Since the opentelemtry (tempo) is an optional feature, it's not certain that this will init tracing with opentelemtry. This is transparent to the caller – i.e. _here_ you don't know (or even care) how the tracing is initialized.

Maybe just rename the function to `tracing::init` and let the implementation handle the details